README.md

```diff
@@ -47,17 +47,9 @@ See also environment variables from lower layers: [`docker-openjdk`](https://git
 
 | Variable                    | solrcore.property variable | java variable                                                | Default                                                      | Comments |
 | --------------------------- | --------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | --------------------------- |
-| SHARDING                    |                      |                                                              | false                                               | if true, configuration folders for shards will be created |
-| NUM_SHARDS                  | shard.count / acl.shard.count |                                                              | 3 | solr6 / solr4 |
-| NUM_NODES           |                  |                                                              | 2                                                        |  |
-| NODE_INSTANCE                  |                         |                                                              | 1                                                   |  |
 | TEMPLATE                  | alfresco.template |                                                              | rerank                                                   |  |
-| REPLICATION_FACTOR              |                     |                                                              | 1                                                        |  |
-| SHARD_IDS                   | shard.instance / acl.shard.instance |                                                              | 0,1                                     | loop over values to create config folders \\ solr6 / solr4 |
-| SHARD_METHOD | shard.method | | DB_ID | solr6 only |
-| SHARD_KEY | shard.key | | cm:creator | solr6 only, used when SHARD_METHOD=PROPERTY |
-| SHARD_RANGE | shard.range | | 0-100000 | solr6 only, used when SHARD_METHOD=DB_ID_RANGE |
 | CORES_TO_TRACK | | | alfresco;archive | loop over values to create config folders \\ solr6 only |
+| CORES_ALFRESCO | | | alfresco | in case of sharded setups, cores to be created on the current host \\ solr6 only |
```

Comment:
if I understand the use of this variable correctly then it means the default value = non-sharded mening "alfresco".
if one wants to create one or more shards, some other value then "alfresco" like "shard" or "alfresco-1" needs to be used? I think this could be better explained what defines a sharded and non-sharded configuration